### PR TITLE
Add real prices

### LINKED ISSUE #108: Add real prices
For now ETH, USD and EUR prices are constants, now we have to make them with real time data.
For example CurrencyDropdown has price prop, which has to be real now, also it is  using currencies, which are constants.

### PR DESCRIPTION
**Pull request by RadoslavDimchev**
Closes #108

I am using [CoinGecko](https://www.coingecko.com/en/api) demo api to get [coin price](https://docs.coingecko.com/v3.0.1/reference/simple-price) and [supported currencies](https://docs.coingecko.com/v3.0.1/reference/simple-supported-currencies). I added required attribution, while we are using demo version below total amount. I am using these calls at CurrencyDropdown component to get real prices and currencies.

### CODE REVIEW

File: src/components/General/CurrencyDropdown.tsx [hunk 1]
```diff
@@ -30,28 +86,43 @@ const CurrencyDropdown = ({ price }: CurrencyDropdownProps) => {
...
       </XStack>
       <Text size={27} weight={'semibold'}>
-        {formatNumbersWithComa(totalPrice)} {currency}
+        {isCurrencyLoading
+          ? LOADING
+          : `${formatNumbersWithComa(totalPrice)} ${currency}`}
       </Text>
+      {/* Attribution required for Demo (Beta)*/}
+      <Text size={11}>Data provided by CoinGecko</Text>
     </YStack>
```
Comment (EmilIvanichkovv) [inline src/components/General/CurrencyDropdown.tsx:126]: We can delete these, no?

File: src/components/General/CurrencyDropdown.tsx [hunk 2]
```diff
@@ -30,28 +86,43 @@ const CurrencyDropdown = ({ price }: CurrencyDropdownProps) => {
     <YStack space={'$2'}>
       <XStack style={{ justifyContent: 'space-between' }}>
         <Text size={15} weight={'semibold'}>
-          {currency}
+          {isCurrencyLoading ? '' : currency}
```
Comment (EmilIvanichkovv) [inline src/components/General/CurrencyDropdown.tsx]:
```suggestion
          {isCurrencyLoading ? '' : getCurrencyLabel(currency)}
```

File: src/components/General/CurrencyDropdown.tsx [hunk 3]
```diff
@@ -30,28 +86,43 @@ const CurrencyDropdown = ({ price }: CurrencyDropdownProps) => {
...
+            height={190}
+            className={'transparent-scrollbar'}
+          >
+            {isSupportedCurrenciesLoading ? (
+              <DropdownMenu.Item label={LOADING} onSelect={() => {}} />
+            ) : (
+              supportedCurrencies.map(currency => (
+                <DropdownMenu.Item
+                  key={currency}
+                  label={currency}
```
Comment (EmilIvanichkovv) [inline src/components/General/CurrencyDropdown.tsx]:
```suggestion
                  label={getCurrencyLabel(currency)}
```

File: src/components/General/CurrencyDropdown.tsx [hunk 4]
```diff
@@ -30,28 +86,43 @@ const CurrencyDropdown = ({ price }: CurrencyDropdownProps) => {
...
+              ))
+            )}
           </DropdownMenu.Content>
         </DropdownMenu>
       </XStack>
       <Text size={27} weight={'semibold'}>
-        {formatNumbersWithComa(totalPrice)} {currency}
+        {isCurrencyLoading
+          ? LOADING
+          : `${formatNumbersWithComa(totalPrice)} ${currency}`}
```
Comment (EmilIvanichkovv) [inline src/components/General/CurrencyDropdown.tsx]:
```suggestion
          : `${formatNumbersWithComa(totalPrice)} ${getCurrencyLabel(currency)}`}
```

File: src/components/General/CurrencyDropdown.tsx
Comment (EmilIvanichkovv) [inline src/components/General/CurrencyDropdown.tsx:1]: Add the following function in `src/utilities.ts` and use it here. Would be even better if we add the corresponding  currency symbol
```ts
export const getCurrencyLabel = (currency: string) => {
  return currency.toUpperCase()
}
```

File: src/constants.ts [hunk 5]
```diff
@@ -174,11 +174,10 @@ export const MAC = 'MacOS'
...
 export const LINUX = 'Linux'
 
-// for now, this will be constant values
-export const CURRENCIES = {
-  USD: 1583.42,
-  EUR: 1323.61,
-}
+export const INITIAL_CURRENCY = 'usd'
+export const LOADING = 'Loading...'
+export const COIN_GECKO_API_KEY = 'CG-FPQgFNTF26FzyCA7vtbaSh5U'
```
Comment (EmilIvanichkovv) [inline src/constants.ts]: Add key to the `.env` file and use it like that:
```suggestion
export const COIN_GECKO_API_KEY = process.env['COIN_GECKO_API_KEY']
```

**Review by EmilIvanichkovv — CHANGES_REQUESTED** [review]

File: src/components/General/CurrencyDropdown.tsx (hunk 1 above)
Comment (RadoslavDimchev) [reply src/components/General/CurrencyDropdown.tsx:126]: I wrote it in the PR desc, we need this while we are using demo api key, but this is not a public software, so we can delete it for now, but I add it just not to forget.

File: src/components/General/CurrencyDropdown.tsx
Comment (RadoslavDimchev) [reply src/components/General/CurrencyDropdown.tsx:1]: Nice for the util func, I forgot about the upper case. For the currency symbol I wrote a local issue on my paper on the desk 😂

File: src/constants.ts (hunk 5 above)
Comment (RadoslavDimchev) [reply src/constants.ts]: gj!

**Review by EmilIvanichkovv — APPROVED** [review]

**Pull request merged**